blocklib/picoscope/Picoscope4000a.hpp

```diff
@@ -12,9 +12,11 @@ class Picoscope4000a : public fair::picoscope::Picoscope<T, acquisitionMode, Pic
 public:
     using super_t = fair::picoscope::Picoscope<T, acquisitionMode, Picoscope4000a<T, acquisitionMode>>;
 
-    Picoscope4000a(gr::property_map props) : super_t(std::move(props)) {}
+    Picoscope4000a(gr::property_map props) : super_t(std::move(props)) {
+        analog_out.resize(8);
+    }
 
-    std::array<gr::PortOut<T>, 8> analog_out;
+    std::vector<gr::PortOut<T>> analog_out{};
```

Comment:
`analog_out{8}` should also work, no? Using `explicit vector( size_type count,
                 const Allocator& alloc = Allocator() );`